presentation/backend/src/presentation-backend/PresentationManager.ts

```diff
@@ -315,9 +323,15 @@ export class PresentationManager {
   constructor(props?: PresentationManagerProps) {
     this._props = props ?? {};
     this._detail = new PresentationManagerDetail(this._props);
+    this.activeLocale = this._props.defaultLocale; // eslint-disable-line deprecation/deprecation
+
+      this._localizationHelper = new LocalizationHelper({ getLocalizedString: props.getLocalizedString ?? getLocalizedStringEN });
```

Comment:
Looks like suggestion broke formatting. Also it looks like it should be `props?.getLocalizedString`